docs/getting-started/architecture-and-components/write-patterns.md

```diff
@@ -0,0 +1,68 @@
+# Writing Data to Feast
+
+Feast uses a [Push Model](getting-started/architecture-and-components/push-vs-pull-model.md) to push features to the online store.
+
+This has two important consequences: (1) communication patterns between the data producer and Feast and (2) feature computation and 
+_feature value_ write patterns to Feast's online store.
+
+Data Producers (i.e., services that generate data) send data to Feast so Feast can write it to the online store. That data can
+be either raw data where Feast computes and store stores the feature values or precomputed feature values.
+
+## Communication Patterns
+
+There are two ways to *_send_* data to the online store: 
+
+1. Synchronously
+   - Using an API call for a small number of entities or a single entity
+2. Asynchronously 
+   - Using an API call for a small number of entities or a single entity
```

Comment:
Can you clarify what you're referring to here? Are there multiple ways to push features to online using feature server API?

Reply:
https://en.wikipedia.org/wiki/Message_passing#Synchronous_versus_asynchronous_message_passing

Do you want me to link to that source?

Reply:
>P.S. of course a user might use client-side async handling, but I don't think that's what you're referring to here.

Actually, this is exactly what I am referring to!

Sounds like I need to make this more explicit if it wasn't obvious.

Reply:
@tokoko take a look, I updated.